### DISCUSSION TITLE
Deleted measurement sets

### DISCUSSION THREAD

**Discussion by joostdecock** [freesewing/freesewing#7135] [Forum]
Hi, stupid.  I button mashed my measurement sets and deleted them.  Anyway to retrieve them?
Thanks

[![User 91344](https://backend3.freesewing.org/users/91344/card)](https://freesewing.org/users/user?id=91344)

**Comment by BenJamesBen** [comment]
Apologies, but I don't think there is a way to recover a deleted Measurements Set.
